docs/src/main/sphinx/sql/create-materialized-view.rst

```diff
@@ -0,0 +1,90 @@
+========================
+CREATE MATERIALIZED VIEW
+========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    CREATE [ OR REPLACE ] MATERIALIZED VIEW
+    [ IF NOT EXISTS ] view_name
+    [ COMMENT string ]
+    [ WITH properties ]
+    AS query
+
+Description
+-----------
+
+Create a new materialized view of a :doc:`select` query. The view is a physical
+manifestation of the query results at time of creation. The data is stored and
+can be referenced by future queries. Since the data is stored directly, these
+queries are typically faster than retrieving data from a view created with the
+same query.
+
+However the data in the materialized becomes out of sync with the underlying
+data in the source tables, when that data changes. The data in the materialized
+view is updated with the :doc:`refresh-materialized-view` statement.
```

Comment:
Refresh the data in the materialized view with the...

Reply:
done